src/main/java/io/strimzi/kafka/metrics/YammerMetricsCollector.java

```diff
@@ -58,27 +58,27 @@ public List<MetricFamilySamples> collect() {
                 Metric metric = entry.getValue();
                 LOG.trace("Collecting Yammer metric {}", metricName);
 
-                String name = metricName(metricName);
+                String prometheusMetricName = metricName(metricName);
                 // TODO Filtering should take labels into account
-                if (!config.isAllowed(name)) {
-                    LOG.info("Yammer metric {} is not allowed", name);
+                if (!config.isAllowed(prometheusMetricName)) {
+                    LOG.info("Yammer metric {} is not allowed", prometheusMetricName);
                     continue;
                 }
-                LOG.info("Yammer metric {} is allowed", name);
+                LOG.info("Yammer metric {} is allowed", prometheusMetricName);
                 Map<String, String> labels = labelsFromScope(metricName.getScope());
-                LOG.info("labels " + labels);
+                LOG.info("labels {} ", labels);
```

Comment:
Should this have some formatting? Or should it be removed? It looks like some leftover debug metric.